apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/DirectRdaLoadApp.java

```diff
@@ -158,4 +177,19 @@ private static RdaLoadOptions readRdaLoadOptionsFromProperties(ConfigLoader opti
     return new RdaLoadOptions(
         jobConfig.build(), grpcConfig, new RdaServerJob.Config(), 0, idHasherConfig);
   }
+
+  /**
+   * Reads aws client settings from the configuration.
+   *
+   * @param options the config options to use
+   * @return the aws config settings
+   */
+  @Nullable
```

Comment:
stared at this for a bit, and can't quite connect the dots....
rdsConfig init'd to null; but I don't see where it actually gets a value?

Reply:
lol, will fix.